data-catalog/base/thriftserver/thriftserver-service.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: thriftserver
+spec:
+  selector:
```

Comment:
This one has the selectors missing
`
  selector:
    name: thriftserver-deployment
```

Reply:
It has

```yaml
selector:
  name: thriftserver
```
Because the `DeploymentConfig` has:
```yaml
metadata:
  name: thriftserver
spec:
  template:
    metadata:
      labels:
        name: thriftserver
```

I've unified the names there. Does it cause problems?